### DISCUSSION TITLE
Get redirected to wrong url while visiting the page

### DISCUSSION THREAD

**Discussion by devhoussam1998** [Z3d0X/filament-fabricator#182] [Q&A] [answered]
when I create a page with name "Home" and slug "/" and click visit link I redirected to another filament page called "Dashboard"

**Comment by Voltra** [comment, marked as answer]
Hi, this is because you probably have kept the basic scaffolding stuff from when Laravel creates a new project for you. Thus that route that renders the welcome view takes priority over your `Page` with slug `/`
